plugin/figure/src/main/java/io/deephaven/figure/FigureWidgetTranslator.java

```diff
@@ -334,9 +354,17 @@ private FigureDescriptor.ChartDescriptor translate(ChartImpl chart) {
                                     // warn about other unsupported series types
                                     errorList.add("OpenAPI presently does not support series of type " + s.getClass());
                                 }
-
-                                // TODO color label size shape
                             } else if (s instanceof AbstractCategoryDataSeries) {
+                                // TODO #3293: Individual point shapes/sizes/labels
+                                // Right now just gets one set for the whole series
+                                AbstractCategoryDataSeries abstractSeries = (AbstractCategoryDataSeries) s;
+                                assignOptionalStringField(abstractSeries.getPointShape(0), clientSeries::setShape,
```

Comment:
Is one special cased in the groovy/py plot apis? I thought that this API was always supposed to take an array of points.

I think instead you want getDefaultValue(), and if not null we will assign that, otherwise if the user assigns "dot square line dot square line" they'll just see "dot dot dot dot dot", and maybe we can send a "sorry, can't support that" error

Reply:
Why is color not in this patch too? Removed from the TODO, but not copied over even for a default? Even if we think that users shouldnt set colors, we might want the ability to use a table column for colors, like format columns do, so it is probably worth partially plumbing...

Reply:
@niloc132 What `getDefaultValue()` are you talking about? 
Color is already handled on line 297.